bridgepoint/prebuild.py

```diff
@@ -1489,8 +1489,11 @@ def __init__(self, metamodel, s_brg):
     @property
     def label(self):
         s_ee = one(self._s_brg).S_EE[19]()
-        return '%s::%s' % (s_ee.Name, self._s_brg.Name)
-    
+        compName = ""
```

Comment:
pyxtuml does not use camel case, use comp_name instead

Reply:
Updated.